content/en/getting_started/integrations.md

```diff
@@ -0,0 +1,178 @@
+
+### Configuring Agent integrations
+
+Configurations are specific to [individual integrations][27]. In the `conf.d` folder at the root of your Agent's configuration directory, there is a folder named `<INTEGRATIONS>.d` for each officially supported Agent integration which contains a sample `conf.yaml.example` that lists all available configuration options for this particular integration.
+
+To activate a given integration:
+
+1. Rename the `conf.yaml.example` file (in the corresponding `<INTEGRATIONS>.d` folder) to `conf.yaml`. 
+2. Update the required parameters inside the newly created configuration file with the values corresponding to your environment.
+3. [Restart the Datadog Agent][31]
+
+**Note**: All configuration files follow the format documented in the [parameters documentation][32].
+
+For example, this is the minimum `conf.yaml` configuration file needed to collect metrics and logs from the [apache integration][33]:
+
+```
+init_config:
+
+instances:
+
+  - apache_status_url: http://localhost/server-status?auto
+
+logs:
+  - type: file
+    path: /var/log/apache2/access.log
+    source: apache
+    sourcecategory: http_web_access
+    service: apache
+  - type: file
+    path: /var/log/apache2/error.log
+    source: apache
+    sourcecategory: http_web_access
+```
+
+To create multiple instances in the same Agent check to monitor two Apache services, create a new instance with a `-` in the `instances:` section:
+
+```
+init_config:
+
+instances:
+
+  - apache_status_url: http://localhost/server-status?auto
+
+  - apache_status_url: http://<REMOTE_APACHE_ENDPOINT>/server-status?auto
+```
+
+### Tagging
+
+Tagging is a key part of filtering and aggregating the data coming into Datadog across many sources. You can assign tags in configuration files, in environment variables, in the UI, or the API, and in DogStatsD. If you define tags in the `datadog.yaml` file, the tags are applied to all of your integrations. Once you've defined the tag in `datadog.yaml`, all new integrations inherit it. If you use a tag environment variable, it applies to all integrations. If you define tags in the corresponding integrations configuration file, it only applies to that specific integration. If you use tags in containers, it applies only to that tag. For more information about tagging, see [Getting started with tags][19].
+
+### Validation
+
+To validate your configuration, [run the Agent's `status` subcommand][34], and look for new configuration under the Checks section.
+
+## Installing multiple integrations
+
+Installing more than one integration is a matter of adding the configuration information to your already existing `conf.yaml` file. Look up the required parameters for the new integration, and add it into your already existing `conf.yaml` file, and then follow the same steps to validate your configuration.
```

Comment:
No not true. It's about writing a new `conf.yaml` in a different `<INTEGRATION2_NAME>.d` folder